webrtc.html

```diff
@@ -11702,6 +11705,19 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
                 <td>The IdP token is invalid.</td>
               </tr>
               <tr>
+                <td><dfn><code>invalid-codecs</code></dfn></td>
+                <td>Invalid or unavailable <code>codecs[]</code>
```

Comment:
May want to restrict this to just "unavailable", since "invalid" is already covered by InvalidModificationError.